README.md

```diff
@@ -1,5 +1,10 @@
-# crocks.js
-`crocks` is a collection of popular *Algebraic Data Types (ADTs)* that are all the rage in functional programming. You have heard of things like `Maybe` and `Either` and heck maybe even `IO`, that is what these are. The main goal of `crocks` is to curate and provide not only a common interface between each type (where possible of course), but also all of the helper functions needed to hit the ground running.
+<img style="display: block; width: 360px; margin: 24px auto;" src="https://cdn.rawgit.com/rstegg/crocks/dfdee636/crocks_js.svg" />
```

Comment:
This should be Markdown. If you need to specify styling, then there may be an issue with the asset.
Also the src will not work for someone viewing offline.
TBH, this is basically just a cartoonish heading. Would rather go back to a simple heading. 
This is an information document, not really a marketing page.